src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddLogCommand.cs

```diff
@@ -0,0 +1,17 @@
+﻿using MediatR;
+
+namespace SFA.DAS.Approvals.Application.BulkUpload.Commands
+{
+    public class BulkUploadAddLogCommand : IRequest<BulkUploadAddLogResult>
+    {
+        public BulkUploadAddLogCommand()
+        {
+        }
+
```

Comment:
The convention is we usually don't add ctors if they are not doing anything

Reply:
Removed